src/mediaelement.js

```diff
@@ -89,7 +124,7 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
     },
 
     seekTo: function (start) {
-        if (start != null) {
+        if (start) {
```

Comment:
Good call, I will revert this. While I was testing, setting media element .currentTime was always going back to zero, and I made this while troubleshooting before I realized it was due to the server (python simple server) and had nothing to do with the library. I should have reverted this and remembered that from [your stack overflow post](http://stackoverflow.com/questions/9563887/setting-html5-audio-position#comment12125431_9564020)